docs/userguide/campaigns.md

```diff
@@ -1,25 +1,125 @@
 # Campaigns
```

Comment:
theres many typos in this file, did it go through typos check or is typos simply allowing them?

Reply:
I forgot install the pre-commit hook in my fork 😆 Will correct.

Reply:
UPDATE: Typos does not check the markdown files, since typos is just a *source code spell checker*. So unless we want to have another linter, we won't have spell checking automatically enabled.

Reply:
Fixed all typos that I found. Please resolve if happy.